src/app/qgsidentifyresultsdialog.cpp

```diff
@@ -1511,6 +1511,8 @@ void QgsIdentifyResultsDialog::contextMenuEvent( QContextMenuEvent *event )
     mActionPopup->addAction( tr( "Copy Attribute Value" ), this, &QgsIdentifyResultsDialog::copyAttributeValue );
     mActionPopup->addAction( tr( "Copy Feature Attributes" ), this, &QgsIdentifyResultsDialog::copyFeatureAttributes );
 
+    mActionPopup->addAction( tr( "Select Features by Attribute" ), this, &QgsIdentifyResultsDialog::selectFeatureByAttribute );
```

Comment:
@nirvn I'd be interested to hear your thoughts on the text for this action. Something about the current version doesn't sound right to me. Maybe something like "Select All Features with Matching Attribute", but that's very verbose...

Reply:
Maybe 'select features {by,matching} attribute value'? Using attribute value here links back to the 'copy attribute value' action above.

Reply:
@nirvn I am not so clear about your suggestion. So, for example, if the attribute and its value are `building_type` and `residential`, then the text will be "select features by building_type",  "Select features residential" or "Select features by building_type = residential"?

Reply:
Ping @nirvn

Reply:
@ismailsunni , sorry, I missed the notification in my inbox (thanks to @nyalldawson for chasing me on this one :) )

I was merely suggesting 'Select Features By Attribute Value' or 'Select Features Matching Attribute Value' as shorter alternatives to what @nyalldawson proposed. But, full disclosure, I'm not a native English speaker (in case that wasn't obvious :wink: )

Reply:
No problem @nirvn. I have updated the string with your first suggestion. I hope it's fine now :). Thanks!